plugins/orchestrator-backend/src/service/ScaffolderService.ts

```diff
@@ -1,7 +1,10 @@
-import { loggerToWinstonLogger, UrlReader } from '@backstage/backend-common';
-import { LoggerService } from '@backstage/backend-plugin-api';
-import { CatalogApi } from '@backstage/catalog-client';
-import { Config } from '@backstage/config';
+import { loggerToWinstonLogger } from '@backstage/backend-common';
```

Comment:
This is the only reference `@backstage/backend-common`. By removing it, we can get rid of the dependency in the `package.json`. 
Still looking for the right way how to do it.

Reply:
I thought We were moving away from Winston logger in our plugins. For example: https://github.com/janus-idp/backstage-plugins/pull/1756/files

Reply:
Let's stick with your approach for now until https://github.com/backstage/backstage/issues/26933 is resolved.